libs/k8s-client/src/BaseApi/PodWithLogStream.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\K8sClient\BaseApi;
+
+use GuzzleHttp\Handler\CurlHandler;
+use GuzzleHttp\HandlerStack;
+use Kubernetes\API\Pod;
+use KubernetesRuntime\Client;
+use Psr\Http\Message\ResponseInterface;
+
+class PodWithLogStream extends Pod
+{
+    public function __construct($namespace = 'default')
+    {
+        $this->client = Client::getInstance();
+        $this->namespace = $namespace;
+    }
+
+    public function readLog(string $namespace, string $name, array $queries = [])
```

Comment:
Takle je to super! :+1:

Az kdyz to ted vidim "na papire" mi doslo, ze jsme vlastne klidne mohli tu metodu pojmenovat jinak, nepretezovat, ale tim bych se ted uz asi netrapil.

Reply:
ok